node-fs-readfile/cat.js

```diff
@@ -0,0 +1,11 @@
+import { readFile } from 'node:fs/promises';
+
+const contentPaths = process.argv.slice(2).map(content =>  `./${content}`);
+
+try {
+  const promises = contentPaths.map(path => readFile(path, 'utf8'))
+  const contents = await Promise.all(promises)
+  console.log(contents.join('\n'));
+} catch (error) {
+  console.log('fail to read content: ', error.message);
```

Comment:
You usually want to log the entire error, not just the message field. That way you will get a stack trace that will help you figure out where things went wrong.